### PR TITLE
Add Stackdriver plugin config

### PR DESCRIPTION
**Pull request by bianpengyuan**
Add Stackdriver plugin config. The config is parsed by StackdriverRootContext.

### CODE REVIEW

File: extensions/stackdriver/config/stackdriver_plugin_config.proto [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+syntax = "proto3";
+
+package stackdriver.config;
+
+message PluginConfig {
+  // Indicates whether the module should act as inbound or outbound reporter.
+  // Difference being:
+  //   * inbound and outbound have different metric types to export.
+  //   * inbound reporter writes access logs.
```
Comment (kyessenov) [inline extensions/stackdriver/config/stackdriver_plugin_config.proto]: Is this still true given there is a boolean `export_access_log` below?

File: extensions/stackdriver/stackdriver.cc [hunk 2]
```diff
@@ -36,14 +36,23 @@ namespace Plugin {
...
 constexpr char kExporterRegistered[] = "registered";
 
 void StackdriverRootContext::onConfigure(
     std::unique_ptr<WasmData> configuration) {
-  // TODO: Add config for Stackdriver plugin, such as reporter kind.
-  UNREFERENCED_PARAMETER(configuration);
+  // Parse configuration JSON string.
+  JsonParseOptions json_options;
+  Status status =
+      JsonStringToMessage(configuration->toString(), &config_, json_options);
```
Comment (kyessenov) [inline extensions/stackdriver/stackdriver.cc]: I think you can use absl::string_view here (`configuration->view()`)

Comment (bianpengyuan) [reply extensions/stackdriver/stackdriver.cc]: Tried but it does not seem to work:
```
extensions/stackdriver/stackdriver.cc:50:7: error: no matching function for call to 'JsonStringToMessage'
      JsonStringToMessage(configuration->view(), &config_, json_options);
      ^~~~~~~~~~~~~~~~~~~
external/com_google_protobuf/src/google/protobuf/util/json_util.h:107:30: note: candidate function not viable: no known conversion from 'StringView' (aka 'absl::string_view') to 'google::protobuf::StringPiece' for 1st argument
PROTOBUF_EXPORT util::Status JsonStringToMessage(
                             ^
external/com_google_protobuf/src/google/protobuf/util/json_util.h:110:21: note: candidate function not viable: requires 2 arguments, but 3 were provided
inline util::Status JsonStringToMessage(StringPiece input,
                    ^
```

File: extensions/stackdriver/config/stackdriver_plugin_config.proto (hunk 1 above)
Comment (bianpengyuan) [reply extensions/stackdriver/config/stackdriver_plugin_config.proto]: Yes, I think we just want to export access log at server side. `export_access_log` controls if server exports access log or not.

File: extensions/stackdriver/stackdriver.cc (hunk 2 above)
Comment (kyessenov) [reply extensions/stackdriver/stackdriver.cc]: Ah, ok. Thanks. I think protobuf hasn't yet pushed StringPiece->string_view update.

**Pull request merged**